controllers/cinder_controller.go

```diff
@@ -686,15 +688,18 @@ func (r *CinderReconciler) reconcileNormal(ctx context.Context, instance *cinder
 		Log.Info(fmt.Sprintf("Deployment %s successfully reconciled - operation: %s", instance.Name, string(op)))
 	}
 
-	// Mirror CinderAPI status' APIEndpoints and ReadyCount to this parent CR
-	instance.Status.APIEndpoints = cinderAPI.Status.APIEndpoints
-	instance.Status.ServiceIDs = cinderAPI.Status.ServiceIDs
-	instance.Status.CinderAPIReadyCount = cinderAPI.Status.ReadyCount
+	// Wait until the data in the StatefulSet is for the current generation
```

Comment:
I like this comment (it's concise and informative). I think it can be repeated for other services as I note below.

Reply:
I think I'll change it to reflect that it's not waiting, it's actually only updating things when the current generation matches, because I don't want to give the impression that this doesn't progress from this point forward if they don't match.